src/test/resources/db/migration/V2018.08.08.14.28.00__add_ach_payment_method_columns.sql

```diff
@@ -0,0 +1,6 @@
+-- Adds ACH specific columns to various tables
```

Comment:
i think it's the `womply-plugin` in the `killbill-resources` repo that this needs to be added to do actually work in environments.

Reply:
@jbarnettwomply: Like in [this PR](https://github.com/OtoAnalytics/killbill-resources/pull/76/files)?